### ISSUE TITLE
Keyboard doesn't open

### ISSUE THREAD

**Issue by Idesmi** [sailfishos-flatpak/main#23]
The virtual keyboard doesn't open when tapping on an input box in AngelFish.

The device is Xperia XA2 running Sailfish 4.2

```
$ flatpak list

default                                org.freedesktop.Platform.GL.default                                       19.08           flathub          user
KDE Application Platform               org.kde.Platform                                                          5.14            flathub          user
Maliit plugin for SailfishOS           org.kde.PlatformInputContexts.MaliitSailfishOS                            5.14            flathub          user
Angelfish Webbrowser                   org.kde.mobile.angelfish                                                  master          kdeapps          user
```

**Comment by felixhesshaw** [comment]
Have the same problem here despite installing the org.kde.PlatformInputContexts.MaliitSailfishOS package

**Comment by rinigus** [comment]
Same here on SFOS 4.2. `dbus-monitor` logs are full of

```
method call time=1635532046.072075 sender=:1.87 -> destination=org.maliit.server serial=56 path=/org/maliit/server/address; interface=org.freedesktop.DBus.Properties; member=Get
   string "org.maliit.Server.Address"
   string "address"
method return time=1635532046.072719 sender=:1.26 -> destination=:1.87 serial=151 reply_serial=56
   variant       string "unix:path=/run/user/100000/maliit-server/dbus-socket,guid=20810e4f9b7059a19d1d5156617c294d"
```

and it sounds like Maliit from Flatpak cannot establish connection with the Maliit server. No idea what changed, could be either on Flatpak itself or SFOS. 

Note that as I couldn't make Flatpak to run for my devices and hence the support for it is reduced until someone who is using it actively can take over and start fixing the bugs.

**Comment by rinigus** [comment]
Found it, `/run/user/100000/maliit-server` was not mounted on Flatpak environment. Should be easy to fix...

**Comment by rinigus** [comment]
Fixed in https://github.com/sailfishos-flatpak/flatpak-runner/commit/6fc0b2e2cf66934a4ad5f692822532fe895f3dc6

Included as a part of flatpak-runner 0.9.1 - currently building at OBS. Closing here
